src/IRutils.cpp

```diff
@@ -599,50 +599,21 @@ namespace irutils {
       char c = unescaped[i];
       switch (c) {
         // ';!-"<>=&#{}() are all unsafe.
-        case '\'':
-          result += F("&apos;");
-          break;
-        case ';':
-          result += F("&semi;");
-          break;
-        case '!':
-          result += F("&excl;");
-          break;
-        case '-':
-          result += F("&dash;");
-          break;
-        case '\"':
-          result += F("&quot;");
-          break;
-        case '<':
-          result += F("&lt;");
-          break;
-        case '>':
-          result += F("&gt;");
-          break;
-        case '=':
-          result += F("&#equals;");
-          break;
-        case '&':
-          result += F("&amp;");
-          break;
-        case '#':
-          result += F("&num;");
-          break;
-        case '{':
-          result += F("&lcub;");
-          break;
-        case '}':
-          result += F("&rcub;");
-          break;
-        case '(':
-          result += F("&lpar;");
-          break;
-        case ')':
-          result += F("&rpar;");
-          break;
-        default:
-          result += c;
+        case '\'': result += F("&apos;"); break;
+        case ';':  result += F("&semi;"); break;
```

Comment:
I have not checked, so this is mostly note to self, unit tests for this?

Reply:
https://github.com/crankyoldgit/IRremoteESP8266/blob/93be2647696c2455b266d8ad9b65dda68cbd59cf/test/IRutils_test.cpp#L406-L420